backendRoot/src/test/java/com/virtuslab/gitmachete/backend/root/GitMacheteRepositoryBuilder_deriveSyncToRemoteStatusTest.java

```diff
@@ -0,0 +1,102 @@
+package com.virtuslab.gitmachete.backend.root;
+
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.reflect.Whitebox;
+
+import com.virtuslab.gitcore.api.IGitCoreBranchTrackingStatus;
+import com.virtuslab.gitcore.api.IGitCoreLocalBranch;
+import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
+
+public class GitMacheteRepositoryBuilder_deriveSyncToRemoteStatusTest {
```

Comment:
I'd also enhance the repo1.sh script to add another remote repo + push one of the branches to that other remote + ensure it's not in sync with the remote so that we can verify the correct message in `status`

Reply:
I added new test for that